src/integrationTest/java/uk/gov/hmcts/reform/ucmc/controllers/CreateClaimControllerTest.java

```diff
@@ -0,0 +1,150 @@
+            .andExpect(status().isOk())
+            .andReturn();
+
+        byte[] responseBody = response.getResponse().getContentAsByteArray();
+
+        SubmittedCallbackResponse callbackResponse = toSubmittedCallbackResponse(responseBody);
+
+        String documentLink = "https://www.google.com";
+        String responsePackLink = "https://formfinder.hmctsformfinder.justice.gov.uk/n9-eng.pdf";
+        LocalDateTime serviceDeadline = LocalDate.now().plusDays(112).atTime(23, 59);
+        String formattedServiceDeadline = formatLocalDateTime(serviceDeadline, DATE_TIME_AT);
+
+        String body = format(
+            "<br />Follow these steps to serve a claim:"
+                + "\n* [Download the sealed claim form](%s) (PDF, 123KB)"
+                + "\n* Send the form, particulars of claim and [a response pack](%s) (PDF, 266 KB) "
+                + "to the defendant by %s"
+                + "\n* Confirm service online within 21 days of sending the form, particulars and response pack, before"
+                + " 4pm if you're doing this on the due day", documentLink, responsePackLink, formattedServiceDeadline);
+
+        assertThat(callbackResponse).isEqualToComparingFieldByField(
+            SubmittedCallbackResponse.builder()
+                .confirmationHeader("# Your claim has been issued\n## Claim number: TBC")
+                .confirmationBody(body)
+                .build());
+    }
+
+    private CallbackRequest toCallbackRequest(Map<String, Object> data) {
+        return CallbackRequest.builder()
+            .caseDetails(CaseDetails.builder().data(data).build())
+            .build();
+    }
+
+    private AboutToStartOrSubmitCallbackResponse postMidEvent(CallbackRequest callbackRequest) throws Exception {
+        MvcResult response = mockMvc
+            .perform(post("/create-claim/mid-event")
+                         .header("authorization", USER_AUTH_TOKEN)
+                         .header("user-id", USER_ID)
+                         .contentType(MediaType.APPLICATION_JSON)
+                         .content(toBytes(callbackRequest)))
+            .andExpect(status().isOk())
+            .andReturn();
+
+        byte[] responseBody = response.getResponse().getContentAsByteArray();
+
+        return toAboutToSubmitResponse(responseBody);
+    }
+
+    private AboutToStartOrSubmitCallbackResponse toAboutToSubmitResponse(byte[] body) throws java.io.IOException {
+        if (body.length > 0) {
+            return mapper.readValue(body, AboutToStartOrSubmitCallbackResponse.class);
+        } else {
+            return null;
+        }
+    }
+
+    private byte[] toBytes(Object o) {
+        try {
+            return mapper.writeValueAsString(o).getBytes();
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private SubmittedCallbackResponse toSubmittedCallbackResponse(byte[] responseBody) throws java.io.IOException {
```

Comment:
These methods could be part of integration test parent class. Also, I believe that helpers methods in test code should handle checked methods, so we don't have to repeat `throws Exception` everywhere it's used.

Just a note for merging later on, both are already done as a part of CMC-388.

Reply:
I'm going to ignore as this will be solved when we play the task to align with 388